.gitlab-ci.yml

```diff
@@ -332,7 +332,6 @@ build:base+async:
 build:vio:
   extends: .build-template:base:dev
   variables:
-    COQ_EXTRA_CONF: "-native-compiler no"
```

Comment:
Why this change?

Reply:
COQ_EXTRA_CONF is for build-template not build-template:base:dev AFAICT